torchqdynamics/progressbar.py

```diff
@@ -0,0 +1,9 @@
+from functools import partial
+
+from tqdm.auto import tqdm as std_tqdm
```

Comment:
I'm really not fan of the notebook progress bar viewer (too big and colorfull IMO, plus it doesn't work well with dark mode), could we switch to have the regular progress bar everywhere (`from tqdm import tqdm`)?

![Screenshot 2023-04-03 at 16 31 40](https://user-images.githubusercontent.com/38159029/229540877-b83d4802-9824-4fd2-9904-e6abe66af5bb.png)